docs/reference/create.md

```diff
@@ -0,0 +1,132 @@
+---
+title: create
+description:
+nav: 204
+---
+
+# create ⚛️
+
+`create` lets you create a React Hook with store API utilities.
+
+```js
+create(stateCreatorFn)
+```
+
+- [Reference](#reference)
+  - [Signature](#create-signature)
+  - [`selector` function](#selector-function)
+  - [`setState` function](#setstate-function)
+  - [`getState` function](#getstate-function)
+  - [`subscribe` function](#subscribe-function)
+  - [`storeApi`](#storeapi)
+- [Usage](#usage)
+  - [Updating state based on previous state](#updating-state-base-on-a-previous-state)
+  - [Updating objects and non-objects in state](#updating-objects-and-non-objects-in-state)
+  - [Subscribing to state updates](#subscribing-to-state-updates)
+- [Troubleshooting](#troubleshooting)
+  - [I’ve updated the state, but the screen doesn’t update](#ive-updated-the-state-but-the-screen-doesnt-update)
+
+## Reference
+
+### `create` Signature
+
+```ts
+create<T>()(stateCreatorFn: StateCreator<T, [], []>): UseBoundStore<StoreApi<T>>
+```
+
```

Comment:
It would be nice if we could teach what a bound hook is.

Reply:
@dai-shi  any other concept that we need to explain? something like store api (utilities), selectors, etc?

Reply:
(Did I miss this comment?)

Yeah, "core concepts" sound nice.

btw, #2138 is ready. I was hoping to merge #2508 in advance to it, but I think either way works.

Reply:
@dai-shi yeah, still working on it I'll try to close reference docs and then work on learn docs